crit-go/scripts/magic_gen.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"github.com/thoas/go-funk"
+	"os"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+func main() {
+
+	if len(os.Args) != 3 {
+		fmt.Println("Usage: magic-gen.py path/to/image.h path/to/magic.json")
```

Comment:
It's no longer `.py` I guess.

Reply:
.. and if you want to call it `magic-gen`, put it into `magic-gen` directory.

Reply:
created a magic-gen directory.